cvat-ui/src/actions/organization-actions.ts

```diff
@@ -220,6 +220,7 @@ export function leaveOrganizationAsync(organization: any): ThunkAction {
             await organization.leave(user);
             dispatch(organizationActions.leaveOrganizationSuccess());
             localStorage.removeItem('currentOrganization');
+            window.location.reload();
```

Comment:
Can we avoid that?

Reply:
I dont think we can avoid reload, we already do it when we change organization to refresh all data in the global state. But now we have `window.location.reload();` in two places related to organizations. I belive this code could be improved with separate action `organizationActions.changeOrganization()` so its all done in one place